home-work-app/src/app/redux/actions/updateMovies.ts

```diff
@@ -0,0 +1,27 @@
+import { IApiResponse } from '../../interfaces/IApiResponse';
+import { IMovieActions } from '../../interfaces/IMovieActions';
+import { MovieService } from '../../services/MovieService';
+import { store } from '../store';
+
+type RequestParams = {
+    sortBy: string,
+    filter?: string[]
+};
+
+export const updateMovieAction: IMovieActions.Update = (queryParams?: RequestParams): any => (dispatch: any) => {
+    if (!store.getState().movieEditor.isUpdated) {
```

Comment:
Why do you need this?
If it is some kind of `loading` flag, I think it's bettre to call it `loading` and dispatch this action inside of `redux-thunk` action creators